{"name": "test_confirm_popup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id: session deleted as the browser has closed the connection\nfrom disconnected: not connected to DevTools\n  (Session info: chrome=140.0.7339.208); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#invalidsessionidexception\nStacktrace:\n\tGetHandleVerifier [0x0xc2c333+65459]\n\tGetHandleVerifier [0x0xc2c374+65524]\n\t(No symbol) [0x0xa4d973]\n\t(No symbol) [0x0xa3cdf0]\n\t(No symbol) [0x0xa5b4af]\n\t(No symbol) [0x0xac0775]\n\t(No symbol) [0x0xa6f904]\n\t(No symbol) [0x0xab9bf6]\n\t(No symbol) [0x0xa8b38e]\n\t(No symbol) [0x0xa8c274]\n\tGetHandleVerifier [0x0xeaeda3+2697763]\n\tGetHandleVerifier [0x0xea9ec7+2677575]\n\tGetHandleVerifier [0x0xc54194+228884]\n\tGetHandleVerifier [0x0xc449f8+165496]\n\tGetHandleVerifier [0x0xc4b18d+192013]\n\tGetHandleVerifier [0x0xc347d8+99416]\n\tGetHandleVerifier [0x0xc34972+99826]\n\tGetHandleVerifier [0x0xc1ebea+10346]\n\tBaseThreadInitThunk [0x0x7570fcc9+25]\n\tRtlGetAppContainerNamedObjectPath [0x0x7761809e+286]\n\tRtlGetAppContainerNamedObjectPath [0x0x7761806e+238]", "trace": "tests\\test_popups.py:31: in test_confirm_popup\n    alert_text = popups_page.trigger_confirm(accept=True)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\npages\\popups_page.py:26: in trigger_confirm\n    self.click_element(self.CONFIRM_BUTTON)\npages\\base_page.py:78: in click_element\n    element = self.find_element(locator, handle_alert)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\npages\\base_page.py:61: in find_element\n    self.handle_alert()\npages\\base_page.py:27: in handle_alert\n    alert = WebDriverWait(self.driver, timeout).until(EC.alert_is_present())\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n..\\..\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\support\\wait.py:129: in until\n    value = method(self._driver)\n            ^^^^^^^^^^^^^^^^^^^^\n..\\..\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\support\\expected_conditions.py:856: in _predicate\n    return driver.switch_to.alert\n           ^^^^^^^^^^^^^^^^^^^^^^\n..\\..\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\remote\\switch_to.py:55: in alert\n    _ = alert.text\n        ^^^^^^^^^^\n..\\..\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\common\\alert.py:58: in text\n    return self.driver.execute(Command.W3C_GET_ALERT_TEXT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n..\\..\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n..\\..\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:232: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id: session deleted as the browser has closed the connection\nE   from disconnected: not connected to DevTools\nE     (Session info: chrome=140.0.7339.208); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#invalidsessionidexception\nE   Stacktrace:\nE   \tGetHandleVerifier [0x0xc2c333+65459]\nE   \tGetHandleVerifier [0x0xc2c374+65524]\nE   \t(No symbol) [0x0xa4d973]\nE   \t(No symbol) [0x0xa3cdf0]\nE   \t(No symbol) [0x0xa5b4af]\nE   \t(No symbol) [0x0xac0775]\nE   \t(No symbol) [0x0xa6f904]\nE   \t(No symbol) [0x0xab9bf6]\nE   \t(No symbol) [0x0xa8b38e]\nE   \t(No symbol) [0x0xa8c274]\nE   \tGetHandleVerifier [0x0xeaeda3+2697763]\nE   \tGetHandleVerifier [0x0xea9ec7+2677575]\nE   \tGetHandleVerifier [0x0xc54194+228884]\nE   \tGetHandleVerifier [0x0xc449f8+165496]\nE   \tGetHandleVerifier [0x0xc4b18d+192013]\nE   \tGetHandleVerifier [0x0xc347d8+99416]\nE   \tGetHandleVerifier [0x0xc34972+99826]\nE   \tGetHandleVerifier [0x0xc1ebea+10346]\nE   \tBaseThreadInitThunk [0x0x7570fcc9+25]\nE   \tRtlGetAppContainerNamedObjectPath [0x0x7761809e+286]\nE   \tRtlGetAppContainerNamedObjectPath [0x0x7761806e+238]"}, "steps": [{"name": "Trigger and accept confirm", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id: session deleted as the browser has closed the connection\nfrom disconnected: not connected to DevTools\n  (Session info: chrome=140.0.7339.208); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#invalidsessionidexception\nStacktrace:\n\tGetHandleVerifier [0x0xc2c333+65459]\n\tGetHandleVerifier [0x0xc2c374+65524]\n\t(No symbol) [0x0xa4d973]\n\t(No symbol) [0x0xa3cdf0]\n\t(No symbol) [0x0xa5b4af]\n\t(No symbol) [0x0xac0775]\n\t(No symbol) [0x0xa6f904]\n\t(No symbol) [0x0xab9bf6]\n\t(No symbol) [0x0xa8b38e]\n\t(No symbol) [0x0xa8c274]\n\tGetHandleVerifier [0x0xeaeda3+2697763]\n\tGetHandleVerifier [0x0xea9ec7+2677575]\n\tGetHandleVerifier [0x0xc54194+228884]\n\tGetHandleVerifier [0x0xc449f8+165496]\n\tGetHandleVerifier [0x0xc4b18d+192013]\n\tGetHandleVerifier [0x0xc347d8+99416]\n\tGetHandleVerifier [0x0xc34972+99826]\n\tGetHandleVerifier [0x0xc1ebea+10346]\n\tBaseThreadInitThunk [0x0x7570fcc9+25]\n\tRtlGetAppContainerNamedObjectPath [0x0x7761809e+286]\n\tRtlGetAppContainerNamedObjectPath [0x0x7761806e+238]\n\n", "trace": "  File \"C:\\Users\\Fr0z3n556\\Desktop\\automation_project\\tests\\test_popups.py\", line 31, in test_confirm_popup\n    alert_text = popups_page.trigger_confirm(accept=True)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Fr0z3n556\\Desktop\\automation_project\\pages\\popups_page.py\", line 26, in trigger_confirm\n    self.click_element(self.CONFIRM_BUTTON)\n  File \"C:\\Users\\Fr0z3n556\\Desktop\\automation_project\\pages\\base_page.py\", line 78, in click_element\n    element = self.find_element(locator, handle_alert)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Fr0z3n556\\Desktop\\automation_project\\pages\\base_page.py\", line 61, in find_element\n    self.handle_alert()\n  File \"C:\\Users\\Fr0z3n556\\Desktop\\automation_project\\pages\\base_page.py\", line 27, in handle_alert\n    alert = WebDriverWait(self.driver, timeout).until(EC.alert_is_present())\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Fr0z3n556\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 129, in until\n    value = method(self._driver)\n            ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Fr0z3n556\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\support\\expected_conditions.py\", line 856, in _predicate\n    return driver.switch_to.alert\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Fr0z3n556\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\remote\\switch_to.py\", line 55, in alert\n    _ = alert.text\n        ^^^^^^^^^^\n  File \"C:\\Users\\Fr0z3n556\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\common\\alert.py\", line 58, in text\n    return self.driver.execute(Command.W3C_GET_ALERT_TEXT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Fr0z3n556\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 458, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Fr0z3n556\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1758991130858, "stop": 1758991131865}], "start": 1758991130858, "stop": 1758991131867, "uuid": "15c97f8e-0e77-4e55-bd12-4c2f9a4a7464", "historyId": "91f1a358be9c8bbc08f1fc5091c9b2ea", "testCaseId": "91f1a358be9c8bbc08f1fc5091c9b2ea", "fullName": "tests.test_popups.TestPopups#test_confirm_popup", "labels": [{"name": "feature", "value": "Popups"}, {"name": "story", "value": "Test confirm popup"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_popups"}, {"name": "subSuite", "value": "TestPopups"}, {"name": "host", "value": "MakuOSPC"}, {"name": "thread", "value": "34040-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_popups"}], "titlePath": ["tests", "test_popups.py", "TestPopups"]}